sinabs/backend/dynapcnn/chips/speck2f.py

```diff
@@ -26,8 +27,30 @@ def get_output_buffer(cls):
         return samna.BasicSinkNode_speck2f_event_output_event()
 
     @classmethod
-    def get_dynapcnn_layer_config_dict(cls, layer: DynapcnnLayer):
-        config_dict = super().get_dynapcnn_layer_config_dict(layer=layer)
+    def get_dynapcnn_layer_config_dict(
+        cls,
+        layer: DynapcnnLayer,
+        layer2core_map: Dict[int, int],
+        destination_indices: List[int],
+    ) -> dict:
+        """Generate config dict from DynapcnnLayer instance
+
+        Parameters
+        ----------
+        - layer (DynapcnnLayer): Layer instance from which to generate the config
+        - layer2core_map (Dict): Keys are layer indices, values are corresponding
+            cores on hardware. Needed to map the destinations.]
```

Comment:
typo: `]` is lost here.
This is clearly a copy-paste mistake. Please go through the code to find other instances.